app/src/main/java/at/vunfer/openrealms/MainActivity.java

```diff
@@ -256,11 +260,15 @@ public void run() {
                                     if (stats.getPlayerHealth() < 1) {
                                         //                                        //this adds the
                                         // defeat screen on top of the game
-                                        ImageView defeatImage = findViewById(R.id.defeat_image);
+                                        defeatImage = findViewById(R.id.defeat_image);
                                         defeatImage.setVisibility(View.VISIBLE);
                                         defeatImage.getParent().bringChildToFront(defeatImage);
+                                        victoryImage = findViewById(R.id.victory_image);
+                                        victoryImage.setVisibility(View.INVISIBLE);
+                                        endTurnButton = findViewById(R.id.end_turn_button);
+                                        endTurnButton.setVisibility(View.INVISIBLE);
                                         Button endTurnButton = findViewById(R.id.end_turn_button);
-                                        endTurnButton.setVisibility(View.GONE);
+                                        endTurnButton.setVisibility(View.INVISIBLE);
```

Comment:
I think there was a merge conflict here, I doubt that we need to find and set the button to Invisible twice.